fixtures/array_initilizer.cpp.sem.uast

```diff
@@ -107,14 +105,10 @@
                            col: 19,
                         },
                      },
-                     ExpressionType: "const char [4]",
-                     ExpressionValueCategory: "LVALUE",
-                     IsLValue: true,
-                     kind: "string_literal",
+                     Format: "",
+                     Value: "\"abc\"",
```

Comment:
Should be unescaped in Semantic (`"abc"`)

Reply:
Fixed, thanks.